lib/src/components/GorgeousHeader/GorgeousHeader.js

```diff
@@ -55,7 +56,7 @@ class GorgeousHeader extends Component {
         <Text style={subtitleTextStyle || styles.subtitleTextStyle}>
           {subtitle}
         </Text>
-        <View style={styles.searchBarStyle}>
+        <View style={[styles.searchBarStyle, searchBarStyle]}>
```

Comment:
You need to change it with this:

`searchBarStyle || styles.searchBarStyle`

Otherwise, it cannot overwrite the search bar's style :)

Reply:
I had left it this way on purpose so if you wanted to make small tweaks to the styles you could since the style values later on in the array override the earlier values. This makes it so you could do something like add shadow without overriding all the style values that exist. Do you still want me to change this?

Reply:
I think you should change it that way because with this way we let the developer to the whole customization and let the decision on them :)

Reply:
I just pushed again with the correction. Let me know if you need anything else.

Reply:
Lovely work! Thank you so much. I will release a new version and inform you.

Reply:
Glad to help. Sounds great